cli/cmd/helm/version.go

```diff
@@ -15,13 +19,42 @@ var versionCmd = &cobra.Command{
 }
 
 func init() {
-	cmd.RootCmd.AddCommand(versionCmd)
+	root.RootCmd.AddCommand(versionCmd)
+	versionCmd.Flags().BoolP("deployed-chart", "d", false, "Option to show deployed tobs helm chart version")
 }
 
 const tobsVersion = "0.4.0"
 
 func version(cmd *cobra.Command, args []string) error {
-	fmt.Printf("Tobs Version: %s\n", tobsVersion)
+	d, err := cmd.Flags().GetBool("deployed-chart")
+	if err != nil {
+		return fmt.Errorf("could not get deployed tobs helm chart version: %w", err)
+	}
+
+	var chartVersion string
+	if d {
+		deployedChart, err := utils.GetDeployedChartMetadata(root.HelmReleaseName, root.Namespace)
+		if err != nil {
+			log.Fatalf("failed to get the deployed chart version %v", err)
+		}
+		if deployedChart != nil {
+			deployedVersion := strings.Split(deployedChart.Chart, "-")[1]
+			chartVersion = fmt.Sprintf("deployed tobs helm chart version: %s", deployedVersion)
+		} else {
+			chartVersion = "unable to find existing tobs deployment"
+		}
+	} else {
+		err = utils.AddUpdateTobsChart(false)
+		if err != nil {
+			return fmt.Errorf("failed to add and update the tobs helm chart version %v", err)
+		}
+		latestChart, err := utils.GetTobsChartMetadata(utils.DEFAULT_CHART)
+		if err != nil {
+			return fmt.Errorf("failed to get latest tobs helm chart version %v", err)
+		}
+		chartVersion = fmt.Sprintf("latest tobs helm chart version: %s", latestChart.Version)
+	}
+
+	fmt.Printf("Tobs CLI Version: %s, %s \n", tobsVersion, chartVersion)
```

Comment:
I wanted to avoid code duplication i.e in this case to short-circuit I need to add them twice in both cases. 
```
	fmt.Printf("Tobs CLI Version: %s, %s \n", tobsVersion, chartVersion)
	return nil
```